bin/templates/cordova/lib/pluginHandlers.js

```diff
@@ -44,11 +37,8 @@ var handlers = {
             }
         },
         uninstall: function (obj, plugin, project, options) {
-            var dest = path.join(obj.targetDir, path.basename(obj.src));
 
-            if (options && options.android_studio === true) {
-                dest = studioPathRemap(obj);
-            }
+            var dest = getInstallDestination(obj);
```

Comment:
Same comment as I made on line 31. Here is a much simpler change:

```diff
@@ -47,7 +47,7 @@ var handlers = {
             var dest = path.join(obj.targetDir, path.basename(obj.src));
 
             if (options && options.android_studio === true) {
-                dest = studioPathRemap(obj);
+                dest = getInstallDestination(obj);
             }
 
             // TODO: Add Koltin extension to uninstall, since they are handled like Java files
```

Reply:
`if (options && options.android_studio === true)` is going to be true always, so there is no point on keeping it